HelpDeskProductPageObject/src/main/java/Admin_Portal/AdminPortalTest1stPage.java

```diff
@@ -134,6 +135,7 @@ public void clickPrioritySection() {
 	}
 	
 	@FindBy(xpath = "/html[1]/body[1]/div[3]/div[1]/section[1]/section[1]/div[1]/div[1]/section[1]/div[1]/div[1]/div[2]/div[1]/div[2]/div[1]/table[1]/tbody[1]/tr[9]/td[2]")
+	
```

Comment:
Change to a better Locator Strategy. Avoid using Absolute Xpath's
Use Java Naming Conventions when naming a variable
Could be named as newPriority instead of NewPriority